content/search-github/github-code-search/using-github-code-search.md

```diff
@@ -40,11 +40,18 @@ For more information about the search syntax of code search, see "[AUTOTITLE](/s
 ## Creating and managing saved searches
 
 1. In the top navigation of {% data variables.product.prodname_dotcom_the_website %}, click the search bar and type `saved:`.
-1. Under the search bar, the "Saved searches" section should now appear. Click {% octicon "plus-circle" aria-hidden="true" %} **Create saved search**.
-1. In the pop-up window, fill out the name you want for your query and the query that you want to save. Click **Create saved search**.
-1. If you click again on the search bar, you can now see your saved search in the "Saved searches" section under the search bar. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
-    - To edit a saved search, in the "Saved searches" section, click {% octicon "pencil" aria-label="The pencil icon" %} to the right of the saved search.
-    - To delete a saved search, click {% octicon "trash" aria-label="The trash icon" %} to the right of the saved search.
+
+1. Under the search bar, in the "Saved queries" section, click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
+
+1. In the pop-up window, type both the name you want for your saved search and the query you want to save.
+
+1. To finish creating your saved search, click **Create saved search**.
+
+1. To see your saved search, click the search bar. Your saved search will be in the "Saved queries" section. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
+
+1. To manage a saved search, type `saved:` in the search bar, then click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
+   To edit a saved search, to the right of the search, click {% octicon "pencil" aria-label="The pencil icon" %}.
+   To delete a saved search, to the right of the search, click {% octicon "trash" aria-label="The trash icon" %}.
```

Comment:
```suggestion
1. Under the search bar, in the "Saved queries" section, click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
1. In the pop-up window, type both the name you want for your saved search and the query you want to save.
1. To finish creating your saved search, click **Create saved search**.
1. To see your saved search, click the search bar. Your saved search will be in the "Saved queries" section. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
1. To manage a saved search, type `saved:` in the search bar, then click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
    - To edit a saved search, to the right of the search, click {% octicon "pencil" aria-label="The pencil icon" %}.
    - To delete a saved search, to the right of the search, click {% octicon "trash" aria-label="The trash icon" %}.
```

Thanks for making those changes @npiyush97! I'll apply a couple final tweaks:

- we can remove those blank lines between the numbered steps.
- I added dashes to the substeps for managing saved searches to create an indented bulleted list. My apologies for missing that earlier!

I'll go ahead and commit these changes, then get this merged 🙂 thanks again for your help maintaining our docs! ✨